agents/plugins/mk_oracle

```diff
@@ -2944,13 +2951,8 @@ do_dummy_sections
 
 #   ---local----------------------------------------------------------------
 
-for sid in $SIDS; do
-    skip=$(skip_sid "$sid")
-    if [ "$skip" == "yes" ]; then
-        logging "[${sid}] [local]" "Skip"
-        continue
-    fi
 
+function sid_actions () {
```

Comment:
- better naming: "sid_actions" -> "execute_queries_for_sid" or similar.

- Line 2959: SC2104: In functions, use return instead of continue.
Please use shellcheck in order to avoid such mistakes